Expecto/Expecto.fs

```diff
@@ -88,23 +88,23 @@ module Tests =
   /// Builds an async test case
   let inline testCaseAsync name test = TestLabel(name, TestCase (Async test,Normal), Normal)
   /// Builds an async test case from a task
-  let inline testCaseTask name test = TestLabel(name, TestCase (Async test,Normal), Normal)
+  let inline testCaseTask name test = TestLabel(name, TestCase (Async (Async.AwaitTask test),Normal), Normal)
```

Comment:
This was the main fix. `testCaseTask` and it's variants were expecting an Async instead of a Task